### PR TITLE
Add Proto.io & MarvelApp

### PR DESCRIPTION
**Pull request by mhrabiee**
Adding Proto.io & MarvelApp

### CODE REVIEW

**Comment by Nightfirecat** [comment]
Please revert "Update with JSONLint"; this change conflicts with the `.editorconfig` file for this repo.

File: sites.json [hunk 1]
```diff
@@ -3575,6 +3575,16 @@
             "mapbox.com"
         ]
     },
+
```
Comment (Nightfirecat) [inline sites.json]: Please remove this trailing space

File: sites.json [hunk 2]
```diff
@@ -4655,14 +4665,24 @@
             "prosper.com"
         ]
     },
-
+
```
Comment (Nightfirecat) [inline sites.json]: Please remove this trailing space

File: sites.json [hunk 3]
```diff
@@ -4655,14 +4665,24 @@
...
+              "proto.io"
+          ]
+    },
+    
     {
          "name": "ProtonMail",
          "url": "https://mail.protonmail.com/account",
          "difficulty": "easy",
          "notes": "Log into your account -> Click Settings -> Scroll Down to the Delete My Account Button -> Type why you are leaving, password, and 2 Factor Authentication code if enabled -> Click Delete",
-         "domains": [
```
Comment (Nightfirecat) [inline sites.json]: Please re-add the spaces deleted here

**Comment by PyroSamurai** [comment]
Use a squash and merge on this, @Nightfirecat

**Comment by mhrabiee** [comment]
Hey Guys,
What else i should have done? I'm kinda beginner at Github. :grin:

**Pull request merged**